### PR TITLE
:recycle: Change liquidator auction parameters

### PR DESCRIPTION
**Pull request by zekiblue**
- Increases Start Price multiplier to %160
- Decreases half-time to 40 minutes

### CODE REVIEW

File: script/utils/Constants.sol [hunk 1]
```diff
@@ -67,3 +67,10 @@ library VAS {
     uint256 internal constant USDC = 10 ** 6;
     uint256 internal constant WETH = 10 ** 8;
 }
+
+library LiquidatorParameters {
+    uint32 internal constant HALF_LIFE_TIME = 40 * 60; // 40 minutes.
+    uint32 internal constant CUTOFF_TIME = 14_400; // 1 hour.
+    uint16 internal constant START_PRICE_MULTIPLIER = 16_000; // 160%. 1.6x
+    uint16 internal constant MIN_PRICE_MULTIPLIER = 6000; // 60%. 6x
```
Comment (j-vp) [inline script/utils/Constants.sol:75]:
```suggestion
    uint16 internal constant MIN_PRICE_MULTIPLIER = 6000; // 60%. 0.6x
```

File: script/utils/Constants.sol [hunk 2]
```diff
@@ -67,3 +67,10 @@ library VAS {
     uint256 internal constant USDC = 10 ** 6;
     uint256 internal constant WETH = 10 ** 8;
 }
+
+library LiquidatorParameters {
+    uint32 internal constant HALF_LIFE_TIME = 40 * 60; // 40 minutes.
+    uint32 internal constant CUTOFF_TIME = 14_400; // 1 hour.
```
Comment (Thomas-Smets) [inline script/utils/Constants.sol:73]:
```suggestion
    uint32 internal constant CUTOFF_TIME = 14_400; // 4 hours.
```

File: script/utils/Constants.sol [hunk 3]
```diff
@@ -67,3 +67,10 @@ library VAS {
     uint256 internal constant USDC = 10 ** 6;
     uint256 internal constant WETH = 10 ** 8;
 }
+
+library LiquidatorParameters {
+    uint32 internal constant HALF_LIFE_TIME = 40 * 60; // 40 minutes.
```
Comment (Thomas-Smets) [inline script/utils/Constants.sol:72]:
```suggestion
    uint32 internal constant HALF_LIFE_TIME = 240; // 40 minutes.
```

File: script/utils/Constants.sol [hunk 4]
```diff
@@ -67,3 +67,10 @@ library VAS {
...
+    uint32 internal constant CUTOFF_TIME = 14_400; // 4 hours.
+    uint16 internal constant START_PRICE_MULTIPLIER = 16_000; // 160%. 1.6x
+    uint16 internal constant MIN_PRICE_MULTIPLIER = 6000; // 60%. 0.6x
```
Comment (j-vp) [inline script/utils/Constants.sol:75]:
```suggestion
    uint16 internal constant MIN_PRICE_MULTIPLIER = 8000; // 80%. 0.8x
```

Comment (j-vp) [reply script/utils/Constants.sol:75]: was previously changed at https://basescan.org/tx/0x2b0316517c8ca2cdb0125af2b6ee441bf04b5e5e8eb0fc2af66d6eedf3cb5908

**Review by j-vp — APPROVED** [review]

**Pull request merged**